src/plugins/orangehrmAdminPlugin/Dao/LocalizationDao.php

```diff
@@ -83,4 +87,79 @@ public function saveI18NLanguage(I18NLanguage $i18NLanguage): I18NLanguage
         $this->persist($i18NLanguage);
         return $i18NLanguage;
     }
+
+    /**
+     * @param I18NTranslationSearchFilterParams $i18NTargetLangStringSearchFilterParams
+     * @return array e.g. [0 => ['id' => 1, 'source' => 'About', 'note' => null, 'target' => '关于']]
+     */
+    public function getNormalizedTranslations(
+        I18NTranslationSearchFilterParams $i18NTargetLangStringSearchFilterParams
+    ): array {
+        $q = $this->getTranslationsQueryBuilderWrapper($i18NTargetLangStringSearchFilterParams)->getQueryBuilder();
+        $q->select(
+            'translation.id',
+            'langString.value AS source',
+            'langString.note AS note',
+            'translation.value AS target',
+        );
+        return $q->getQuery()->execute();
```

Comment:
maybe could return just `array_values` from here? or do you need in this specific array format elsewhere
```suggestion
        return array_values($q->getQuery()->execute());
```

Reply:
Done